skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -342,11 +345,34 @@ def test_keys_deleter_no_keys(self):
         self.assertFalse(msa.has_keys())
 
     def test_keys_deleter_with_keys(self):
-        msa = TabularMSA([RNA('UUU'), RNA('AAA')], key=str)
+        msa = TabularMSA([RNA('UUU'), RNA('AAA')], minter=str)
         self.assertTrue(msa.has_keys())
         del msa.keys
         self.assertFalse(msa.has_keys())
 
+    def test_get_cached_minter_str(self):
```

Comment:
Another interesting test to add would be testing that an `msa` initialized with `keys` raises an error when `minter` is accessed.